app/frontend/src/components/Datepicker.tsx

```diff
@@ -40,10 +50,9 @@ export default function Datepicker({
             animateYearScrolling={true}
             autoOk
             className={className}
-            clearable
             disableToolbar
             error={error}
-            format="DD.MM.YYYY"
+            format={getLocaleFormat()}
             fullWidth
             helperText={helperText}
             id={id}
```

Comment:
```suggestion
            id={id}
            KeyboardButtonProps={{
              'aria-label': CHANGE_DATE,
            }}
```
Adapted from the [date picker doc's](https://material-ui.com/components/pickers/) example code. You can export the `CHANGE_DATE` variable and put it in this file I think, since it's specific to this component only.

This will then hopefully make my other suggested change in the test work and make the button more accessible in the process!